bin/start

```diff
@@ -6,13 +6,13 @@ Test the secrets-provider-for-k8s image. This script sets up a Conjur cluster in
 container and a secrets-provider-for-k8s init container. Finally it tests that the outcome is as expected (for example,
 in the vanilla flow it verifies that the Conjur secret was provided to the app container via the environment.
 
-Usage: ./bin/test_integation [options]
+Usage: ./bin/start [options]
     --docker      Run the integation tests in a docker container. This is helpful when running the test in a machine that
                   doesn't have "oc" or "kubectl" installed on it.
 
-    --demo        Run a full end-to-end demo with the pet-store demo. This demo deploys an environment with an app container
-                  that needs credentials to a DB from Conjur. After running "./bin/test_integation --demo" run "oc get routes"
-                  and go to "<the exposed route>/pets" in a browser. The entry "Mr. Sidecar" should be displayed on the screen.
+    --dev         Run the development environment. By default the environment is run against a Conjur OSS deployment on GKE.
+
+    --reload      Run reload to rebuild and redeploy changes to a local cluster.
```

Comment:
i would emphasize what this flag saves from doing. for example, don't create a DAP cluster

Reply:
`--reload      Run reload to rebuild and redeploy changes done in Secrets Provider to a local cluster without redeploying DAP/Conjur.`

sound good?

Reply:
yes, thanks